toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -48,14 +48,22 @@ func RecordBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, b
 	}
 
 	prebuiltSRPMs := make(map[string]*pkggraph.PkgNode)
+	prebuiltDeltaSRPMS := make(map[string]*pkggraph.PkgNode)
 	builtSRPMs := make(map[string]*pkggraph.PkgNode)
 	unbuiltSRPMs := make(map[string]*pkggraph.PkgNode)
 	unresolvedDependencies := make(map[string]bool)
 
 	buildNodes := pkgGraph.AllBuildNodes()
 	for _, node := range buildNodes {
+
+		// Convert the below to a switch statement. A node can be a delta if it was build or cached. If it was cached
+		// we used the cached rpm. If it is not cached that means it was built and we discard the delta rpm.
```

Comment:
Instead of leaving a comment, can we convert it now?

Reply:
I think that was a TODO that turned out to not be a better solution. The switch ended up being more complicated than the original version. I've properly cleaned up that comment.